explainer.md

```diff
@@ -335,13 +403,68 @@ not yet resolved. This causes work that is still needed to become synchronous,
 enabling the idle-until-urgent pattern.
 
 
-##### DisplayLockContext.updateAndCommit()
+#### DisplayLockContext.updateAndCommit()
 
 This operation combines the effects of an `update()` and a `commit()` calls:
 * It causes the element to be co-operatively updated.
 * When the update is finished, the element is committed resulting in visual
   updates to appear on screen.
 
+
+#### Element activation
+
+When an element is locked,
+there are some actions in the page that might require the element to get unlocked and be rendered to work properly.
+
+If the browser needs to *activate* a locked element
+or an element that has one or more locked ancestors,
+in order to make the activated element be visible,
+it will commit all locked elements in the ancestor chain of the element to be activated,
```

Comment:
I'm not sure if it's correct but maybe

Will commit all locked elements in the inclusive ancestor chain which have the `activatable` flag set to `true`

But I'm not sure if the bubbling jumps over or stops at activatable=false locked elements.

Reply:
We haven't considered this case I think. I added an agenda item to discuss at the
weekly meeting.

Reply:
If any of its inclusive ancestor is locked & non-activatable, the element can't be activated, since even if we unlock everything else, it still won't be visible.